jena-db/jena-tdb2/src/main/java/org/apache/jena/tdb2/sys/SystemTDB.java

```diff
@@ -158,6 +158,15 @@ static public void init() {}
     /** Size of Node lookup miss cache. */
     public static final int NodeMissCacheSize       = 1000;
 
+    /** Size of Node to NodeId cache for prefixes. */
+    public static final int PrefixNode2NodeIdCacheSize    = intValue("PrefixNode2NodeIdCacheSize", 250);
+
+    /** Size of NodeId to Node cache. for prefixes.*/
```

Comment:
dangling period in the middle of sentence :grimacing:

Reply:
Done